assembly/assembly-wsmaster-war/src/main/java/org/eclipse/che/api/deploy/WsMasterModule.java

```diff
@@ -305,7 +305,9 @@ private void configureMultiUserMode(
     if (OpenShiftInfrastructure.NAME.equals(infrastructure)
         || KubernetesInfrastructure.NAME.equals(infrastructure)) {
       install(new ReplicationModule(persistenceProperties));
-
+      bind(
```

Comment:
Can you elaborate why this is specific to the infra implementation?

Reply:
Because `BrokerService` is kubernetes specific component that is bound only for Kubernetes and OpenShift infrastructures. Does it make sense?